build.gradle

```diff
@@ -55,7 +55,7 @@ dependencies {
     final javadocJDKFiles = files(((URLClassLoader) ToolProvider.getSystemToolClassLoader()).getURLs())
     compileOnly(javadocJDKFiles)
     testCompile(javadocJDKFiles)
-    compile 'org.freemarker:freemarker:2.3.18'
+    compile 'org.freemarker:freemarker:2.3.23'
```

Comment:
I install this branch in my local repository (it is not in the artifactory due to #80) and use it in the latest [gatk-master](https://github.com/broadinstitute/gatk/tree/2f2eb73be34e822ebea84b9f116c506768c625a1). Using `./gradlew clean gatkDoc` no error/warning was thrown.

Regarding nice features in this version that may be useful for barclay, and my main idea behind this change, are the built-in handling of special numbers ("INF", "-INF", "NaN") to check argument ranges (I could add this PR or a future PR to populate "NaN" instead of "NA" for some arguments).